misc/FFNx.toml

```diff
@@ -705,6 +705,10 @@ ff8_fix_uv_coords_precision = true
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ff8_external_music_force_original_filenames = false
 
+# Use PlayStation icons for gamepad buttons
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ff8_use_playstation_icons = false

```

Comment:
@julianxhokaxhiu are you ok with the name?

Reply:
Can the playstation icons be replaced using the external texture approach?

Reply:
Yes, if the option ff8_use_playstation_icons is enabled, it will use the icons.tex texture, which is overridable via external texture

Reply:
Ah nice. So would it make use to name it something like `ff8_use_gamepad_icons`? Thinking about the possibility to override the tex file with an Xbox layout for example via a custom mod. Although by default will use the PSX icons, keeping the name generic might also benefit the mod guide for eg. What do you think?

I know it's a minor thing but I'm open for suggestions in case. Thanks!